app/models/queries/FormQueries.py

```diff
@@ -0,0 +1,52 @@
+from app.models import Forms
+from app.models import Files
+
+
+
+class FormQueries():
+    def select_all(self):
+        '''This method is to select all the forms stored in the database'''
+        try:
+            forms = Forms.select()
+            return forms
+        except Exception as e:
+            return False
```

Comment:
Instead of returning false please return None

Reply:
Also please have a look at the way that the other queries are commented, we want to generate documentation and it only works when the text is commented in a particular style.